R/error-helpers.R

```diff
@@ -444,14 +444,14 @@ abort_w_message <- function(msg) {
   abort(msg, .subclass = dm_error_full("w_message"))
 }
 
-# no table zoomed, but 'dm_insert_zoomed_tbl()' called ---------------------------------
+# no table zoomed, but 'dm_insert_zoomed()' called ---------------------------------
 
 abort_no_table_zoomed <- function() {
   abort(error_no_table_zoomed(), .subclass = dm_error_full("no_table_zoomed"))
 }
 
 error_no_table_zoomed <- function() {
-  "`dm_insert_zoomed_tbl()` only works for zoomed `dm`"
+  "`dm_insert_zoomed()` only works for zoomed `dm`"
```

Comment:
```suggestion
  "`dm_insert_zoomed()` only works for zoomed `dm`."
```

Here and everywhere.